src/Products/PlonePAS/tools/memberdata.py

```diff
@@ -238,11 +223,17 @@ def _getPlugins(self):
 InitializeClass(MemberDataTool)
 
 
-@implementer(IManageCapabilities)
-class MemberData(BaseMemberData):
+# FIXME: MemberData should probably be a more specific adapter
+# instead of a subclass.
```

Comment:
If the Interface is the same, there is no need to subclass IMember just for the sake of being more specific, so IMO this is perfectly fine.

Reply:
Thanks for the review. I'll remove the the FIXME.